content/docs/getting-started/installation/installationwizard/src/static/js/commands.js

```diff
@@ -31,4 +31,5 @@ const veleroNote = 'If Velero is enabled, please add the respective credentials
 const podmonNote = 'Uncomment tolerations under node property, if CSM for Resiliency and CSI Driver pods monitor are enabled in the generated YAML';
 const authorizationNote = 'Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately';
 const topologyNote = 'If Node Topology is enabled, make sure topology configmap is created';
-const replicationNote = 'If Replication is enabled, ensure the Replication CRDs are installed and secrets are created.';
\ No newline at end of file
+const replicationNote = 'If Replication is enabled, ensure the Replication CRDs are installed and secrets are created.';
+const powermaxCredNote = 'The CSI PowerMax Reverse Proxy ConfigMap will no longer be supported as of May 30, 2025 and completely deprecated  by January 31, 2026. Please switch to using the CSI PowerMax Mount Credentials before these dates to prevent any potential disruptions. Migration steps are available for both <a href="/csm-docs/docs/getting-started/upgrade/kubernetes/powermax/helm/" target="_blank" rel="noopener noreferrer">Helm</a> and <a href="/csm-docs/docs/getting-started/upgrade/kubernetes/powermax/operator/" target="_blank" rel="noopener noreferrer">Operator</a>.'
```

Comment:
@sharmilarama do we want dates only or release versions or both?

Reply:
Both - we should follow the convention from all our deprecation notices in our home page https://dell.github.io/csm-docs/docs/